spynnaker/pyNN/utilities/utility_calls.py

```diff
@@ -40,7 +40,11 @@ def convert_param_to_numpy(param, no_atoms):
 
```

Comment:
I still believe that the no_atoms > 1 check is no longer needed, as even in PyNN 0.7, if the return value is an array, it will haveattr __iter__, thus it will be picked up in the next check.  A simple test would confirm this...